tools/clang/unittests/SPIRV/FileTestFixture.cpp

```diff
@@ -123,7 +124,7 @@ void FileTest::runFileTest(llvm::StringRef filename, Expect expect) {
   ASSERT_EQ(result.status(), effcee::Result::Status::Ok);
 
   // Run SPIR-V validation for successful compilations
-  if (expect != Expect::Failure) {
+  if (!noValidation && expect != Expect::Failure) {
```

Comment:
Nit: double negative! It'd be easier if the function takes `bool runValidation` (with default value of `true`).
Then this condition will be 
```
if(runValidation && expect != Expect::Failure) {
```

Reply:
Yep. Will do in the next PR.